default.env

```diff
@@ -75,3 +75,15 @@ export KLOG_VERBOSITY='1'
 # Example: 5d
 # Required: no
 export IMAGE_TAG_EXPIRATION='6h'
+
+# A valid Github App ID
+# Required: yes
+PAC_GITHUB_APP_ID=''
+
+# A valid private key encoded in base64 format, it's downloaded from GitHub App
+# Required: yes
+PAC_GITHUB_APP_PRIVATE_KEY=''
+
+# A valid Webhook secret configured in GitHub App
+# Required: no
+PAC_GITHUB_APP_WEBHOOK_SECRET=''
```

Comment:
should these be exported like other env vars?

and are `PAC_GITHUB_APP_PRIVATE_KEY` and `PAC_GITHUB_APP_WEBHOOK_SECRET` env vars used somewhere?

Reply:
yes, they should be exported in https://github.com/openshift/release/blob/master/ci-operator/step-registry/redhat-appstudio/e2e/redhat-appstudio-e2e-commands.sh

Reply:
got it , I will remove the three . these env vars are exported in this PR https://github.com/openshift/release/pull/42622/files

Reply:
fixed.